src/applications/yellow-ribbon/components/SearchResultsPage/index.js

```diff
@@ -30,15 +30,7 @@ export const SearchResultsPage = ({
         {/* Pre-form content */}
         <p className="vads-l-col--12 medium-screen:vads-l-col--9">
           Participating school information on this page is valid for the current
-          academic year, from August 2019 through July 2020. To view schools for
-          the previous academic year,{' '}
-          <a
-            href="https://www.benefits.va.gov/GIBILL/yellow_ribbon/yrp_list_2018.asp"
-            rel="noreferrer noopener"
-          >
-            view 2018 - 2019 rates
-          </a>
-          .
+          academic year, from August 2019 through July 2020.
```

Comment:
We could use `moment('08-01-2019').format('MMMM YYYY')` for this to show `August 2019`. Same for `July 2020`.